docs/source/samples/calibration/calibration_reader.rst

```diff
@@ -27,24 +27,17 @@ Here's theoretical calculation of the focal length in pixels:
 
 .. math::
 
-  focal_length_in_pixels = image_width_in_pixels * 0.5 / tan(HFOV * 0.5 * PI/180)
```

Comment:
Unrelated but let's also add to docs thr script to calculate full FOV for any camera (horizontal vertical diagonal), discussed on slack

Reply:
good point, thanks